src/algebra/module/linear_map.lean

```diff
@@ -53,7 +53,7 @@ linear map
 -/
 
 open function
-open_locale big_operators
+open_locale big_operators pointwise
```

Comment:
I'd prefer putting
```
section pointwise
open_locale pointwise

end pointwise
```
around the new lemmas, which would also help point future contributors to put pointwise lemmas in that section.